src/main/java/ch/wisv/chue/HueService.java

```diff
@@ -88,14 +99,23 @@ public void loadEvent(HueEvent event, int duration, String... lightIdentifiers)
      * @param lightIdentifiers the lights to strobe
      */
     public void strobe(int millis, String... lightIdentifiers) {
-        hueFacade.strobe(millis, getLightIdentifiers(lightIdentifiers));
+        if (!hueFacade.bridgeAvailable()) {
+            throw new EventNotExecutedException("Hue bridge is not available");
+        }
+
+        try {
+            hueFacade.strobe(millis, getLightIdentifiers(lightIdentifiers));
+        } catch (BridgeUnavailableException e) {
+            e.printStackTrace();
```

Comment:
Bad idea. And would this actually happen, since you check that the bridge is available?